python/test/Ice/custom/Custom.py

```diff
@@ -0,0 +1,85 @@
+# **********************************************************************
+#
+# Copyright (c) 2003-2018 ZeroC, Inc. All rights reserved.
+#
+# This copy of Ice is licensed to you under the terms described in the
+# ICE_LICENSE file included in this distribution.
+#
+# **********************************************************************
+
+import numpy
+import array
+
+
```

Comment:
We need some tests to check that error handling is correct:

- if the factory function returns None
- if the factory function raises an exception (ValueError is raised for example when re-shaping a numpy array)